portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,524 @@
+                </a>
+              </div>                
+            </div>
+            <!--/.Card-->
+          </div>
+
+          <div class="col-lg-4 col-md-6 col-sm-12 mb-5">
+            <!--Card-->
+            <div class="card wow fadeIn mb-5 pb-1 h-100" data-wow-delay="0.2s">
+              <!-- Card image -->
+              <div class="view overlay">
+                <img class="card-img-top" src="img/projects/messaging.png" alt="Card image cap">
+                <a href="#!">
+                  <div class="mask rgba-white-slight waves-effect waves-light"></div>
+                </a>
+              </div>
+              <!--Card content-->
+              <div class="card-body">
+                <!--Title-->
+                <h4 class="card-title">Chatting and Image sharing App</h4>
+                <!--Text-->
+                <p class="card-text ">A MEAN stack app which allows users to share posts containing images and CRUD fnctionality with autoorization and authentication.</p>
+                <a href="https://github.com/arjan-bal/mean-chatting-app" target="_blank" class="btn btn-info mdb-color darken-4   waves-effect waves-light" style=" color: #f5f5f5;">
+                  <i class="fab fa-github mr-2 white-text"></i>Github
+                </a>
+              </div>                
+            </div>
+            <!--/.Card-->
+          </div>
+        </div>
+
+      </section>
+    </div>
+
+    <!-- Contact Section -->
+    <hr>
+
+    <div class="container text-center">
+      <section id="contact" class="section mb-5">
+        <h2 class="text-center text-uppercase my-5 pt-5 wow fadeIn" data-wow-delay="0.2s">Get in <strong>Touch</strong></h2>
+          <div class="row pr-lg-5">
+            
+            <div class="col-md-7">
+              <div class="view">
+                <img src="img/contact.png" class="img-fluid" alt="smaple image">
+              </div>
+            </div>
+
+            <div class="col-md-5 d-flex align-items-center">
+              <div class="p-4">
+                  
+                  <p>I'm available on Facebook, LinkedIn and Quora. Mails and LinkedIn are the best way to communicate though.
+                    You can reach out to me on on my coding profiles too.</p>
+                  <a href="https://www.linkedin.com/in/arjan-s-bal" target="_blank">                      
+                      <button type="button" class="btn btn-primary btn-rounded mx-0 waves-effect waves-light">Connect</button>                      
+                  </a>
+              </div>
+
+          </div>
+          </div>
+      </section>
+    </div>
+
+    <footer class="page-footer text-center font-medium mt-4 wow fadeIn">
+      <!--Footer-->    
+      <!--Copyright-->
+      <div class="footer-copyright py-4">
+          © 2020 Copyright | 
+          <a href="https://github.com/sonumeewa" target="_blank">Arjan <strong>Singh Bal</strong> </a>
```

Comment:
sonumeewa?

Reply:
He's a college friend, I've used the footer from one of his projects. Forgot to remove the hyperlink 😅